components/inputs/test/input-number.vdiff.js

```diff
@@ -91,6 +100,26 @@ describe('d2l-input-number', () => {
 		await expect(elem).to.be.golden();
 	});
 
+	it('right side tooltip', async() => {
+		const elem = await fixture(
+			html`<d2l-input-number label="Label" value=1 max=0 label-hidden></d2l-input-number>`,
+			{ viewport: { width:600, height:80 } }
+		);
+		await focusElem(elem);
+		await oneEvent(elem, 'd2l-tooltip-show');
```

Comment:
Since focusing is going to trigger the tooltip, I'd suggest not `await`-ing the focus. It could be possible that while waiting for the focus to happen, the tooltip event actually fires before the next line has a chance to `await` it.

```suggestion
		focusElem(elem);
		await oneEvent(elem, 'd2l-tooltip-show');
```